src/Standards/BestIt/Sniffs/Commenting/MethodDocSniff.php

```diff
@@ -0,0 +1,293 @@
+     */
+    const MESSAGE_SPACING_AFTER = 'Additional blank lines found at end of doc comment';
+
+    /**
+     * Code for missing spacing between descriptions
+     *
+     * @var string
+     */
+    const CODE_SPACING_BETWEEN = 'SpacingBetween';
+
+    /**
+     * Message for missing spacing between descriptions
+     *
+     * @var string
+     */
+    const MESSAGE_SPACING_BETWEEN = 'There must be exactly one blank line between descriptions in a doc comment';
+
+    /**
+     * Code for missing spacing before the tags
+     *
+     * @var string
+     */
+    const CODE_SPACING_BEFORE_TAGS = 'SpacingBeforeTags';
+
+    /**
+     * Message for missing spacing before the tags
+     *
+     * @var string
+     */
+    const MESSAGE_SPACING_BEFORE_TAGS = 'There must be exactly one blank line before the tags in a doc comment';
+
+    /**
+     * Code for missing short description (Summary)
+     *
+     * @var string
+     */
+    const CODE_MISSING_SHORT = 'MissingShort';
+
+    /**
+     * Message for missing short description (Summary)
+     *
+     * @var string
+     */
+    const MESSAGE_MISSING_SHORT = 'Missing short description in doc comment';
+
+    /**
+     * Code for not capitalized short description
+     *
+     * @var string
+     */
+    const CODE_SHORT_NOT_CAPITAL = 'ShortNotCapital';
+
+    /**
+     * Message for not capitalized short description
+     *
+     * @var string
+     */
+    const MESSAGE_SHORT_NOT_CAPITAL = 'Doc comment short description must start with a capital letter';
+
+    /**
+     * Code for not Capitalized long description
+     *
+     * @var string
+     */
+    const CODE_LONG_NOT_CAPITAL = 'LongNotCapital';
+
+    /**
+     * Message for not Capitalized long description
+     *
+     * @var string
+     */
+    const MESSAGE_LONG_NOT_CAPITAL = 'Doc comment long description must start with a capital letter';
+
+    /**
+     * Code for empty lines before the short description
+     *
+     * @var string
+     */
+    const CODE_SPACING_BEFORE_SHORT = 'SpacingBeforeShort';
+
+    /**
+     * Message for empty lines before the short description
+     *
+     * @var string
+     */
+    const MESSAGE_CODE_SPACING_BEFORE = 'Doc comment short description must be on the first line';
+    #endregion
+
+    /**
+     * A list of tokenizers this sniff supports.
+     *
+     * @var array
+     */
+    public $supportedTokenizers = ['PHP', 'JS'];
+
+    /**
+     * Returns an array of tokens this test wants to listen for.
+     *
+     * @return array
+     */
+    public function register()
+    {
+        return [T_DOC_COMMENT_OPEN_TAG];
+    }
+
+    /**
+     * Processes this test, when one of its tokens is encountered.
+     *
+     * @param PHP_CodeSniffer_File $phpcsFile The file being scanned.
+     * @param int $stackPtr The position of the current token in the stack passed in $tokens.
+     *
+     * @return void
+     */
+    public function process(PHP_CodeSniffer_File $phpcsFile, $stackPtr)
```

Comment:
how do you handle inline decls?
```
/** @var $foo Foobar **/`
/** @var Foobar $foo **/`
```

whats about add a _nextSibling_ check for a _method token_ in doc comment scope?